privacyidea/lib/resolvers/LDAPIdResolver.py

```diff
@@ -426,7 +426,18 @@ def _get_uid(entry, uidtype):
                 # In order to ensure backwards compatibility for user mappings,
                 # we strip the curly braces from objectGUID values.
                 # If we are using ldap3 <= 2.4.1, there are no curly braces and we leave the value unchanged.
-                uid = uid.strip("{").strip("}")
+                try:
+                    uid = convert_column_to_unicode(uid).strip("{").strip("}")
+                except UnicodeDecodeError as e:
+                    # in some weird cases we sometimes get a byte-array here
+                    # which resembles a uuid. So we just convert it to one...
+                    # We might run into endian-issues here depending on what
+                    # ldap3/AD returns.
+                    log.debug('Found a byte-array as uid ({0!s}), trying to '
```

Comment:
Should we increase the log level, since this is s.th. that *should* not happen. So that if it happens it is not hidden by the debug-log-level.

Reply:
Sounds good to me, will change to `warning`